gsplat/project_gaussians.py

```diff
@@ -57,6 +57,7 @@ def project_gaussians(
         - **cov3d** (Tensor): 3D covariances.
     """
     assert block_width > 1 and block_width <= 16, "block_width must be between 2 and 16"
+    assert (quats.norm(dim=-1) - 1 < 1e-6).all(), "quats must be normalized"
```

Comment:
It looks good to me. But I want @vye16 to verify the change. See #153 for the reason of why we want to merge this PR.

Reply:
Likely minor, but are there performance concerns here? This seems ~equivalent to a `torch.cuda.synchronize()` call.

Reply:
I think this is fine.

Reply:
ok!